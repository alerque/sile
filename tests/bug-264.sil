\begin[papersize=a7,class=book]{document}
\nofolios
Footnote\skip[height=170pt plus 50pt]
marker.\footnote{Same page\hfill\break as marker?}
\end{document}
