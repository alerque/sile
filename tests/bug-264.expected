Set paper size 	209.7637818	297.6377985
Begin page
Mx 	37.4104
My 	40.7760
Set font 	Gentium Plus;10;400;;normal;;LTR
T	41 82 82 87 81 82 87 72	(Footnote)
New page
Mx 	49.3669
My 	42.0943
T	80 68 85 78 72 85	(marker)
Mx 	79.4060
T	17	(.)
Mx 	81.6960
My 	38.9156
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	20	(1)
Mx 	29.3669
My 	235.7565
Set font 	Gentium Plus;9;400;;normal;;LTR
T	20	(1)
Mx 	33.5901
T	17	(.)
Mx 	53.6511
T	54 68 80 72	(Same)
Mx 	75.8312
T	83 68 74 72	(page)
Mx 	29.3669
My 	247.7565
T	68 86	(as)
Mx 	39.3643
T	80 68 85 78 72 85	(marker)
Mx 	66.3994
T	34	(?)
End page
Finish
